browser/dasher/controlpanel.js

```diff
@@ -567,6 +567,9 @@ export default class ControlPanel {
             // Top of the structure; descend structure with same settings.
             return settings;
         }
+        if (settings === undefined){
```

Comment:
I think it was a loading/timing issue, but when running tests and launching Dasher using chromium, I was getting error messages- 'Can not read 'main' from null object' (or something similar to that). I added this check to see if settings was undefined, but not sure if there are unintended consequences..

Reply:
From our conversation on Slack ...

To get to the bottom of it, maybe you could try these things:

-   Remove the .catch(() => {}) from load_from_browser. That could be silently discarding an error message about failure to open or read from the database.
-   Check or assert that event.target.result isn’t undefined on line 710.
-   See if there’s a Puppeteer option to delete any persistent storage on start-up. Could headless Chrome be persisting the indexedDB stores between executions?